code/_core/obj/item/clothing/back/storage/bluespace/_bluespace.dm

```diff
@@ -27,7 +27,7 @@
 	icon = 'icons/obj/item/clothing/back/satchel/bluespace.dmi'
 
 	dynamic_inventory_count = MAX_INVENTORY_X*4
-	container_max_size = SIZE_4
+	container_max_size = SIZE_4*3
```

Comment:
You should be using the size defines for this (SIZE_5,SIZE_6,SIZE_7) instead of directly multiplying it.